src/main/java/seedu/address/model/Calendar.java

```diff
@@ -0,0 +1,70 @@
+package seedu.address.model;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import javafx.collections.ObservableList;
+import seedu.address.model.event.Event;
+import seedu.address.model.event.UniqueEventList;
+
+public class Calendar implements ReadOnlyCalendar {
+    private final UniqueEventList events;
+
+    {
+        events = new UniqueEventList();
+    }
```

Comment:
We mirrored AddressBook class, I've copied over the explanation this time. Explanation:
    /*
     * The 'unusual' code block below is a non-static initialization block, sometimes used to avoid duplication
     * between constructors. See https://docs.oracle.com/javase/tutorial/java/javaOO/initial.html
     *
     * Note that non-static init blocks are not recommended to use. There are other ways to avoid duplication
     *   among constructors.
     */